core/src/main/java/com/oxeanbits/forecastchart/core/ui/component/CombinedChartComponent.kt

```diff
@@ -31,17 +41,24 @@ class CombinedChartComponent(context: Context) : LinearLayout(context), Anvil.Re
             v(CombinedChart::class.java) {
                 size(MATCH, 500)
                 margin(10)
-                init{
-                    combinedChart = Anvil.currentView()
+                init {
+                    this.combinedChart = Anvil.currentView()
+                    this.combinedChart?.let {
```

Comment:
An alternative version, solving the anti arrow pattern:

```suggestion
val combinedChart =  this.combinedChart ?: return
val expectedData = this.expectedData ?: return
val actualData = this.actualData ?: return
ForecastChart.createForecastChart(context, combinedChart, expectedData, actualData)
```